src/app/components/MatchItem/MatchItem.tsx

```diff
@@ -4,12 +4,13 @@ import classes from './MatchItem.module.css';
 
 type MatchItemProps = {
   match: Match;
+  onClick?: () => void;
```

Comment:
Would it be useful, to move the type to the types.ts file? to collect all types at one single file?

Reply:
I don't think this type should be outsourced, because it is specific for this component. You would do the same with `propTypes`.